src/python/director/drakevisualizer.py

```diff
@@ -200,6 +200,44 @@ def __init__(self, name, geom, polyData, parentTransform):
             self.polyDataItem.shadowOn()
 
 
+class LidarSource(TimerCallback):
```

Comment:
Hmm... okay, but means I have to create additional methods to forward the start/stop calls. BTW, the code in `pointcloudlcm.py` (which I was copying) is doing an ugly combination of *both*...